src/helpers/context.js

```diff
@@ -1,16 +1,22 @@
 // @flow
 import JubilationTheme from '../jubilation-theme';
 
+function scale(x: number): number { return x; }
+scale.domain = function domain(): number[] { return [100, 0]; };
+scale.range = function domain(): number[] { return [0, 100]; };
+
+export const defaultContext: JubilationContext = {
+  theme: JubilationTheme,
+  xScale: scale,
+  yScale: scale,
+  xRange: [0, 300],
+  yRange: [0, 100],
+  addDomain() {},
+  removeDomain() {},
+  update() {},
+};
+
+
 export default function getContext(context: JubilationContext): JubilationContext {
-  if (context) return context;
```

Comment:
What was the reasoning behind this refactor?  Make `defaultContext` a singleton?